src/reformat_libyuv.c

```diff
@@ -52,7 +53,7 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
     }
 
     if ((rgb->chromaUpsampling != AVIF_CHROMA_UPSAMPLING_AUTOMATIC) && (rgb->chromaUpsampling != AVIF_CHROMA_UPSAMPLING_FASTEST)) {
-        // libyuv uses its own upsampling filter. If the enduser chose a specific one, avoid using libyuv.
+        // If the enduser chose a specific one, avoid using libyuv.
```

Comment:
If we delete the first sentence "libyuv uses its own upsampling filter.", we need to change "one" to "upsampling filter" in the second sentence.

It would still be good to provide a reason. I guess you deleted "libyuv uses its own upsampling filter." because it is incorrect?

Maybe we can say something like the following:

> If the enduser chose a built-in upsampling filter or the best quality upsampling, avoid using libyuv.

Reply:
Done.
I added reasons of why we avoided libyuv in these cases.